osh/hub/settings_local.ci.py

```diff
@@ -111,7 +111,9 @@
     'amqps://umb-broker06.stage.api.redhat.com:5671']
 
 UMB_CLIENT_CERT = '/etc/osh/hub/msg-client-osh.pem'
-UMB_TOPIC_PREFIX = 'topic://VirtualTopic.eng.OpenScanHub.scan'
+# UMB_TOPIC_PREFIX can be a single string or a []-defined list of strings
+# Every UMB message wil be sent to all the defined UMB_TOPIC_PREFIX
```

Comment:
```suggestion
# UMB_TOPIC_PREFIX can be a single string or a collection of strings.
# Every UMB message will be sent to all defined UMB topic prefixes.
```
* The code you committed will also work for other collections like tuples or sets.  Besides, the second sentence was incomplete.
* I also suggest to enable a spell checker in your editor for future contributions.

Reply:
Fixed and agree.